vcr/holder/interface.go

```diff
@@ -60,6 +60,9 @@ type Wallet interface {
 
 	// IsEmpty returns true if the wallet contains no credentials at all (for all holder DIDs).
 	IsEmpty() (bool, error)
+
+	// Delete deletes a credential from the wallet
+	Delete(ctx context.Context, subjectDID did.DID, id ssi.URI) error
```

Comment:
Already in master under the method `Remove()`

Reply:
Agree, will remove